spec/persister/finders_spec.rb

```diff
@@ -58,22 +58,6 @@
     end.to(raise_error(expected_error))
   end
 
-  it "checks that we recognize first argument vs passed kwargs" do
-    # This passes nicely
-    lazy_vm = persister.vms.lazy_find({:name => "name"}, :ref => :by_name)
-    expect(lazy_vm.reference.full_reference).to eq(:name => "name")
-
-    # TODO(lsmola) But this fails, since it takes the whole hash as 1st arg, it should correctly raise invalid format
-    expected_error = "Finder has missing keys for index :manager_ref, missing indexes are: [:ems_ref]"
-    expect do
-      persister.vms.lazy_find({:name => "name", :ref => :by_name})
```

Comment:
So this actually works now because the unmatched `:name` is landing in the kwargs